software/evcc.yml

```
@@ -0,0 +1,12 @@
+name: "evcc"
+website_url: "https://evcc.io/"
+source_code_url: "https://github.com/evcc-io/evcc"
+description: "Extensible EV Charge Controller and home energy management system."
+licenses:
+  - MIT
+platforms:
+  - deb
+  - Docker
+  - Go
+tags:
+  - Miscellaneous
```

```suggestion
  - Internet of Things (IoT)
```
Based on the description and other items like it, I think it'd be better suited for this tag. Otherwise, feel free to ignore.

---

That fits mich better, thanks!
(Kinda forgot we even had that as a tag)